views/candidates/edit.ejs

```diff
@@ -10,7 +10,41 @@
     <label>Work culture preferences</label>
     <input type="text" placeholder="Values, benefits..." name="work_culture_preferences" value = "<%= candidate.work_culture_preferences %>">
     <button type="submit" >Update</button>
-  </form>
+  </form>   -->
   
-
+  <div class="container-fluid h-100">
+    <div class="row justify-content-center align-items-center"></div>
+    <div class="main-body">
+      <hr>
+      <h1 class="font-weight-light">Edit your candidate profile:</h1>    
```

Comment:
We need to add more fields for this form (same for edit job offer) according to the questionnaire but i guess we can do that later